src/mlpack/bindings/R/mlpack/LICENSE

```diff
@@ -0,0 +1,3 @@
+YEAR: 2020
```

Comment:
I am not sure, If we should change this to `YEAR: 2021` and  if we should, then we can also automate the process using CMake. This might work here:

```CMAke
string(TIMESTAMP LICENSE_YEAR "%Y")
```
And I guess static code analysis checks is still failing.

Reply:
Good idea to dynamically set the license year to the most recent.

Reply:
Nice idea!  I added code to do this in cd48b339d.

The static code analysis issue seems to be a false positive; I tried a workaround in 889c27b4.